app/script/view_model/bindings/ConversationListBindings.js

```diff
@@ -0,0 +1,53 @@
+/*
+ * Wire
+ * Copyright (C) 2017 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+'use strict';
+
+// show scroll borders
+ko.bindingHandlers.bordered_list = (function() {
+  const calculate_borders = _.throttle(function($element) {
+    if ($element) {
+      window.requestAnimationFrame(function() {
+        const archive_column = $($element).parent();
+        if (($element.height() <= 0) || !$element.is_scrollable()) {
+          return archive_column.removeClass('left-list-center-border-bottom conversations-center-border-top');
+        }
+
+        archive_column.toggleClass('left-list-center-border-top', !$element.is_scrolled_top());
+        archive_column.toggleClass('left-list-center-border-bottom', !$element.is_scrolled_bottom());
+      });
+    }
+  }
+  , 100);
```

Comment:
Feel free to add the rule and clean it up. A simple timeout is not really that magic. I am es6 migrating, not rewriting all of our code.